salt/modules/mac_brew.py

```diff
@@ -256,6 +257,7 @@ def refresh_db():
 
         salt '*' pkg.refresh_db
     '''
+    salt.utils.pkg.clear_rtag(__opts__)
```

Comment:
Given that this is clearly a pattern that's established across package modules, it would be good to document the reason that this needs to be here and what's happening for future reference. Maybe even automate a test explicitely for it?